### PR TITLE
Make local run not interfere with local projects

### PR DESCRIPTION
**Pull request by krzyzacy**

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -137,6 +137,16 @@ def start(self):
         env.update(self.env_files)
         env.update(self.env)
         self.install_prerequisites()
+        # Do not interfere with the local project
+        try:
+            if env['PROJECT']:
+                check(['gcloud', 'config', 'set', 'project', env['PROJECT'])
+            elif os.environ.get('JOB_NAME') and 'ci-' in os.environ['JOB_NAME']:
```
Comment (spxtr) [inline scenarios/kubernetes_e2e.py]: `pull-kubernetes-e2e-gci-gce` :)

Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: 😢

Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: changed to `startswith`

**Comment by krzyzacy** [comment]
PTAL

**Comment by spxtr** [comment]
I'm not a huge fan of magic environment variables, but I'm also not sure if that's normal for these files.

**Comment by krzyzacy** [comment]
I feel like, everyone should have PROJECT set, so I'm fine with just keep the first statement.

File: scenarios/kubernetes_e2e.py [hunk 2]
```diff
@@ -137,6 +137,12 @@ def start(self):
         env.update(self.env_files)
         env.update(self.env)
         self.install_prerequisites()
+        # Do not interfere with the local project
+        try:
+            if env['PROJECT']:
+                check(['gcloud', 'config', 'set', 'project', env['PROJECT']])
+        except subprocess.CalledProcessError:
+            pass
```
Comment (spxtr) [inline scenarios/kubernetes_e2e.py]: Should we print a warning here?

Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: done

**Pull request merged**